packages/smooth_app/lib/pages/personalized_ranking_page.dart

```diff
@@ -90,7 +90,10 @@ class _PersonalizedRankingPageState extends State<PersonalizedRankingPage> {
     return Scaffold(
       key: _scaffoldKey,
       bottomNavigationBar: BottomNavigationBar(
-        backgroundColor: Theme.of(context).bottomAppBarColor.withAlpha(255),
+        backgroundColor: Theme.of(context)
```

Comment:
Doesn't sound relevant, as it looks like you're setting backgroundcolor to its default value with 0 transparency (alpha=255).
Once we set the `BottomNavigationBarTheme`, that will be the default for the whole app, without having to explicitly set it on `BottomNavigationBar`

Reply:
Yes, there was only one strange phenomenon, the navBar had a strange gradient, the lower part of the navBar was gray. To fix this, I assigned an alpha. But that's not good for the main nav bar, as it should be a bit see-through.
![Navbar](https://user-images.githubusercontent.com/39344769/105504272-dc590180-5cc7-11eb-98d6-0096e692f328.PNG)